### ISSUE TITLE
v5.1.3 doesn't have a `dist/index.js` file

### ISSUE THREAD

**Issue by TooTallNate** [tunnckoCore/parse-function#118]
And thus is broken:

```
000 ▲ 102 -4 → ~ (master) $ npm i parse-function
+ parse-function@5.1.3
added 9 packages in 2.807s

000 ▲ 102 -4 → ~ (master) $ node
> require('parse-function')
Error: Cannot find module 'parse-function'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
```

**Comment by tunnckoCore** [comment]
ahhh, damn, knew it! sorry but cant fix it immediately, im not ot desktop ;/

later this day. try >=5.1.0<5.1.3 ?

sorry about that again.

**Comment by TooTallNate** [comment]
Ya I already pinned locally, was just letting you know 😛

**Comment by tunnckoCore** [comment]
cool, which is the last working version?

**Comment by TooTallNate** [comment]
5.1.2

**Comment by tunnckoCore** [comment]
Okey, it's published at v5.1.5
